### ISSUE TITLE
history delete doesn't work

### ISSUE THREAD

**Issue by AndreKR** [chrisant996/clink#452]
I was on Clink 1.4.22.something. I have a history items 1 to 136, as shown by `history`. I typed `history delete 136` and if I remember correctly it showed something like "Deleted item 136". But a subsequent `history` would still show the item.

I then updated to Clink 1.4.23.5fc3fc. I now have a history with items 1 to 139. If I type `history delete 136` now I get "Unable to delete history item 136.".

I now typed `history clear` and that worked.

**Comment by chrisant996** [comment]
Sharing your `clink_history` file might help reveal something.  You can email it to the address in my github profile, if you don't want to post it here.

**Comment by chrisant996** [comment]
It was broken for 25 months, since v1.1.43.  Apparently `history delete #` isn't used very often.

**Comment by codermonkey55** [comment]
@chrisant996 , I just ran into this myself; glad to see it has been fixed. Any estimates on when the new version with the fix will be published?

**Comment by chrisant996** [comment]
Oops, I successfully caught the COVID (first time; fully vaxxed, incl. bivalent booster).

I'm not going to publish a new version of Clink until after I'm better.  There's no sense risking regressions due to brain fog, or risking inability to apply fixes in a timely manner if a regression were to occur.

**Comment by codermonkey55** [comment]
> It's been that way for 25 months, and there's a workaround, so it doesn't seem urgent. Clink v1.4.24 will be published in the next few days, when some other changes are ready.

What's the workaround?

**Comment by codermonkey55** [comment]
No worries, like you said, it's not urgent. Regarding your COVID, hope that's truly the case and it's a mild/short case.

**Comment by chrisant996** [comment]
I published v1.4.24.

My COVID continues to fade. So I decided I may as well publish the update. It seems I'm one of the lucky ones that has a super mild/fast response to this strain. No complaints here! My wife got hit _hard_ for many days a few months ago, but we managed to isolate so it didn't spread to me. I've even been able to keep working full time (from home), and the impact is less than most colds. 🤷‍♂️ I attribute it to the vaccines/boosters, and to my generally strong immune system. Phew.
